site.ts

```diff
@@ -105,13 +110,72 @@ const style = (id, appearance) => `
         padding-bottom: 4px;
     }
     .description {
-        text-align: center;
         font-size: 14px;
         margin-top: .75rem;
         margin-bottom: .75rem;
-        color: ${appearance?.descriptionTextColor || 'black'};
+        color: ${appearance?.descriptionTextColor || '#4b4b52'};
+    }
+    .rating-options {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-evenly;
+        margin-top: .5rem;
+        margin-bottom: .5rem;
+    }
+    .ratings-number {
+        background-color: #e0e2e8;
+        font-size: 14px;
+        border-radius: 6px;
+        border: 1px solid #e0e2e8;
+        padding: 8px;
+    }
+    .ratings-number:hover {
+        cursor: pointer;
+        filter: brightness(.75);
+    }
+    .rating-options-buttons {
+        display: flex;
+        justify-content: space-evenly;
+    }
+    .rating-options-emoji {
+        display: flex;
+        justify-content: space-evenly;
+    }
+    .ratings-emoji {
+        font-size: 16px;
+        background-color: transparent;
+        border: none;
+    }
+    .ratings-emoji:hover {
+        cursor: pointer;
+    }
+    .emoji-svg:hover {
+        fill: ${appearance?.ratingButtonHoverColor || 'coral'};
+    }
+    .rating-text {
+        display: flex;
+        flex-direction: row;
+        font-size: 12px;
+        justify-content: space-between;
+        margin-top: .5rem;
+        margin-bottom: .5rem;
+        color: #4b4b52;
+    }
+    .rating-section {
+        margin-bottom: .5rem;
     }
 `
+export const getSessionRecordingUrl = (posthog) => {
```

Comment:
this is built into posthog-js now https://github.com/PostHog/posthog-js/blob/master/src/posthog-core.ts#L1484C5-L1484C27

Reply:
yayyyyy